compiler/src/iree/compiler/Codegen/Common/GPU/GPUDistribute.cpp

```diff
@@ -32,12 +32,18 @@ struct GPUDistributePass : public GPUDistributeBase<GPUDistributePass> {
         getEntryPoint(funcOp)->getWorkgroupSize().value(),
         [&](Attribute attr) { return llvm::cast<IntegerAttr>(attr).getInt(); });
 
+    // TODO: Thread through subgroup size everywhere.
+    std::optional<llvm::APInt> maybeSubgroupSize =
+        getEntryPoint(funcOp)->getSubgroupSize();
+    // TODO: Don't hard code kCudaWarpSize here.
+    int64_t subgroupSize =
+        maybeSubgroupSize ? maybeSubgroupSize->getSExtValue() : 32;
```

Comment:
I forgot I left 32 hardcoded here. At a minimum I think it would be better to make a constexpr at the top of this file and I can refactor the way subgroup size is specified as a follow up. Can I push directly to this branch?

Reply:
@qedawkins yes.

Reply:
@qedawkins Well, since it already passed all the tests, it would be better to have your change as a separate PR. I will commit it now.

Reply:
That's fine too, I was planning a follow up anyway.